### PR TITLE
Syncing and cleanup of chainspec.toml values

### PR DESCRIPTION
**Pull request by sacherjj**
Syncing of 1.5.6 settings into chainspec.toml.
Added documentation in chainspec.toml around dynamic gas_prices.
Corrections to default values on local chainspec.

### CODE REVIEW

**Review by darthsiroftardis — APPROVED** [review]
LGTM

**Comment by cspramit** [comment]
section transaction.v1 in the chainspec , is missing

**Review by cspramit — APPROVED** [review]
LGTM

File: resources/production/chainspec.toml [hunk 1]
```diff
@@ -336,6 +336,26 @@ pay = 10_000
 
 
 [vacancy]
+# The cost of a transaction is based on a multiplier. This allows for economic disincentives to misues of the network.
```
Comment (ACStone-MTS) [inline resources/production/chainspec.toml]:
```suggestion
# The cost of a transaction is based on a multiplier. This allows for economic disincentives for misuse of the network.
```

**Pull request merged**